frontend/apps/ern-reconnet/vite.config.js

```diff
@@ -5,17 +5,17 @@ export default defineConfig({
   plugins: [vue()],
   base: "",
   server: {
-    proxy: require("../dev-proxy.config"),
+    proxy: require("../../dev-proxy.config"),
   },
   css: {
     preprocessorOptions: {
       scss: {
         additionalData: `
-          @import "../../molgenis-viz/src/styles/palettes.scss";
-          @import "../../molgenis-viz/src/styles/mixins.scss";
+          @import "../../libs/molgenis-viz/src/styles/palettes.scss";
```

Comment:
@davidruvolo51 can't we use the produced style.css?

Reply:
By default, sass palettes, mixins, and variables aren't compiled into the post-build css file. It is possible to define sass exports, but I wanted to keep the file sizes as lightweight as possible.